Failed to compile for update Vlang version "v up"

**V version: 0.1.23 519f299** 
**OS** 
Windows

**What did you do?**
I just do "v up" in my cmd with admin mode after install v language

**What did you expect to see?**
I expect to see the "last version is allready install" 
 
**What did you see instead?**
The error message returned is:

LINK : fatal error LNK1104: impossible d'ouvrir le fichier 'C:\languages\v\v.exe'
V failed to build itself
Failed to compile - Create an issue at 'https://github.com/vlang' and tag '@emily33901'!
Exiting from error

--- a/tools/vup.v
+++ b/tools/vup.v
@@ -1,3 +1,5 @@
+module main
+
 import os
 
 fn main() {
@@ -11,7 +13,7 @@ fn main() {
 		if os.exists( v_backup_file ) {
 			os.rm( v_backup_file )
 		}
-		os.mv_by_cp('$vroot/v.exe', v_backup_file) or { panic(err) }
+		os.mv('$vroot/v.exe', v_backup_file)
 		s2 := os.exec('"$vroot/make.bat"') or { panic(err) }
 		println(s2.output)
 	} $else {

